server/db/User.js

```diff
@@ -29,22 +27,26 @@ module.exports.deleteUserByUserId = (userId) => {
 }
 
 module.exports.deleteUserByUsername = (username) => {
-  return db.query(`DELETE FROM users where username = '${username}'`);
+  return db.query(`DELETE FROM users where username='${username}'`);
 }
 
 module.exports.createUser = (username, password, profile) => {
   return db.query(`INSERT INTO users (username, password, profile) VALUES ('${username}', '${password}', '${profile}')`);
 }
 
-module.exports.editUserProfile = (userId, profile) => {
-  return db.query(`UPDATE users SET profile = '${profile}' WHERE id = ${userId};`);
+module.exports.editUserProfile = (username, profile) => {
+  return db.query(`UPDATE users SET profile = '${profile}' WHERE username='${username}';`);
```

Comment:
Is there a reason why we changed it from userid to username? Maybe create one for both...